### DISCUSSION TITLE
Rustdesk docker as HA Add-on

### DISCUSSION THREAD

**Discussion by djansen1987** [alexbelgium/hassio-addons#1631] [Ideas]
Dear,

great addons in this repository, i have no idea how much work it is but i think below remote desktop server is a great addition to the series already created here. i also started an item on reddit about it: https://www.reddit.com/r/rustdesk/comments/1gpoej1/rustdesk_home_assistant_addon_docker/

documentation to docker installation: https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/docker/

**Comment by alexbelgium** [comment]
Hi, thanks - I don't really have time to add new addons for the moment. However, there is already a remote desktop server with Guacamole that you could try

**Reply by djansen1987** [reply]
Wow thst does even look better. Thanks will look into that
